workers/unity/Assets/Gdk/Physics/Tests/Editmode/Systems/TickSystemTests.cs

```diff
@@ -0,0 +1,35 @@
+using Improbable.Gdk.TransformSynchronization;
+using NUnit.Framework;
+using Unity.Entities;
+
+namespace Improbable.Gdk.EditmodeTests.TransformSynchronization.Systems
+{
+    [TestFixture]
+    public class TickSystemTests
+    {
+        [Test]
+        public void GlobalTick_gets_incremented_in_OnUpdate()
+        {
+            // Construct a world
+            // The "using" ensures the world will get destroyed (along with its systems)
+            using (var world = new World("test-world"))
+            {
+                // Add system
+                var tickSystem = world.GetOrCreateManager<TickSystem>();
+
+                // Make assertion against initial state of system
+                Assert.AreEqual(0, tickSystem.GlobalTick);
+
+                // Execute system
```

Comment:
Nitpick - the comments in this file are just rewording of the line below them.

Reply:
Removing them from here, but thinking to keep them in the document that has a copy of this test